modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/SystemObjectDefinitionMetadataModelListener.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.object.system.model.listener;
+package com.liferay.object.internal.model.listener;
```

Comment:
Do we want to get rid of the system part of the java package?

Other option could be to leave it as it was before with internal like `com.liferay.object.internal.system.model.listener`

Reply:
Mi though was that as already exists a package for model listeners, makes sense to me to put this one there. 
Also, exists the package `internal.system` which contains the `UserSystemObjectDefinitionMetadata` and adding another package just for the model listener seems weird to me. 
But maybe is just me and makes total sense 😅